.github/workflows/ci.yml

```diff
@@ -91,54 +90,6 @@ jobs:
               type=gha,scope=${{github.base_ref}}-frontend
             cache-to: type=gha,mode=max,scope=${{github.ref_name}}-frontend
 
-
-    frontend_container_arm64:
-      name: Build arm64 Frontend Container
-      needs: frontend_lint_test_build
-      runs-on: ubuntu-latest
-      permissions:
-        contents: read
-        packages: write
-      steps:
```

Comment:
This seems to remove the arm builds for some reason?

Reply:
No, the arm builds are just fully moved to "after E2E has passed".